packages/eslint-config/eslint-config.json

```diff
@@ -0,0 +1,66 @@
+{
+  "extends": [
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "plugin:jsdoc/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "env": {
+    "es6": true
+  },
+  "globals": {
+    "globalThis": "readonly",
+    "BigInt": "readonly",
+    "assert": "readonly",
+    "harden": "readonly",
+    "lockdown": "readonly",
+    "Compartment": "readonly",
+    "StaticModuleRecord": "readonly"
+  },
+  "rules": {
+    "implicit-arrow-linebreak": "off",
+    "function-paren-newline": "off",
+    "arrow-parens": "off",
+    "strict": "off",
+    "prefer-destructuring": "off",
+    "no-else-return": "off",
+    "no-console": "off",
```

Comment:
And this makes the implicit inclusion of `"console"` doubly confusing. Do they by default both include `"console"` and prohibit it?

Reply:
That is likely. The difference would likely be in the shape of the warning or error, from “I don’t know about that” to “I know about that but you shouldn’t use it”

Reply:
> “I know about that but you shouldn’t use it”

Yes, if this config is omitted, eslint emits a warning (`no-console`) whenever you try to use `console.*`.